platform/linux-generic/pktio/dpdk.c

```diff
@@ -436,6 +440,97 @@ static inline int mbuf_to_pkt(pktio_entry_t *pktio_entry,
 	return (i > 0 ? i : -1);
 }
 
+static inline int packet_parse(void *l3_hdr, odp_bool_t *l3_proto_v4,
+			       uint8_t *l4_proto)
```

Comment:
Confusing function name. We already have packet_parse_*() functions in odp_packet.c which do the input packet parsing. In my opinion e.g. check_proto() would be a better name.

Reply:
OK